EndpointForm.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="uk">
```

Comment:
Краще `en` бо `uk` позначає що сайт на українській мові
```suggestion
<html lang="en">
```